Clinical-Genomics/scout
# Edit case and variant's comments

Fix #1918.

This PR makes it possible for users to edit their own comments. Editable comments:
- **Case comments**: comment text is editable
- **Variant comments** : comment text and comment's level (global -> specific and specific -> global) is editable

**How to test**:
1. Install on stage
1. Go to a case with comments from some other user
1. Make sure you can't edit those comments from other users
1. Insert a comment for a case and then change it
1. Insert a comment for a variant and then change it (modify also the comment level)

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] tests executed by EM
